### ISSUE TITLE
./train.sh result window

### ISSUE THREAD

**Issue by gpqls** [jkjung-avt/hand-detection-tutorial#33]
> I0429 22:25:22.272610 140074595292928 basic_session_run_hooks.py:262] loss = 19.405396, step = 1
INFO:tensorflow:global_step/sec: 0.300812
I0429 22:30:54.705383 140074595292928 basic_session_run_hooks.py:692] global_step/sec: 0.300812
INFO:tensorflow:loss = 4.1986165, step = 101 (332.433 sec)
I0429 22:30:54.705876 140074595292928 basic_session_run_hooks.py:260] loss = 4.1986165, step = 101 (332.433 sec)
INFO:tensorflow:Saving checkpoints for 178 into ssd_mobilenet_v2_egohands/model.ckpt.
I0429 22:35:13.434082 140074595292928 basic_session_run_hooks.py:606] Saving checkpoints for 178 into ssd_mobilenet_v2_egohands/model.ckpt.

It looked as if it was coming out this way, suddenly these words came out.

> Instructions for updating:
tf.py_func is deprecated in TF V2. Instead, there are two
    options available in V2.
    - tf.py_function takes a python function which manipulates tf eager
    tensors instead of numpy arrays. It's easy to convert a tf eager tensor to
    an ndarray (just call tensor.numpy()) but having access to eager tensors
    means `tf.py_function`s can use accelerators such as GPUs as well as
    being differentiable using a gradient tape.
    - tf.numpy_function maintains the semantics of the deprecated tf.py_func
    (it is not differentiable, and manipulates numpy arrays). It drops the
    stateful argument making all functions stateful.

Also, these words were printed.

> W0429 22:35:21.032854 140071483201280 coco_evaluation.py:133] Ignoring detection with image id 644676416 since it was previously added
WARNING:tensorflow:Ignoring ground truth with image id 1194511846 since it was previously added
W0429 22:35:21.066672 140071474808576 coco_evaluation.py:82] Ignoring ground truth with image id 1194511846 since it was previously added
WARNING:tensorflow:Ignoring detection with image id 1194511846 since it was previously added
W0429 22:35:21.066817 140071474808576 coco_evaluation.py:133] Ignoring detection with image id 1194511846 since it was previously added
WARNING:tensorflow:Ignoring ground truth with image id 195028185 since it was previously added
W0429 22:35:21.101681 140071474808576 coco_evaluation.py:82] Ignoring ground truth with image id 195028185 since it was previously added
WARNING:tensorflow:Ignoring detection with image id 195028185 since it was previously added
W0429 22:35:21.101828 140071474808576 coco_evaluation.py:133] Ignoring detection with image id 195028185 since it was previously added
INFO:tensorflow:Evaluation [50/500]

After [500/500], I was able to see some sort of evaluation.

> INFO:tensorflow:Evaluation [500/500]
I0429 22:35:38.560459 140074595292928 evaluation.py:167] Evaluation [500/500]
creating index...
index created!
INFO:tensorflow:Loading and preparing annotation results...
I0429 22:35:39.022960 140071474808576 coco_tools.py:109] Loading and preparing annotation results...
INFO:tensorflow:DONE (t=0.00s)
I0429 22:35:39.024285 140071474808576 coco_tools.py:131] DONE (t=0.00s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=0.10s).
Accumulating evaluation results...
DONE (t=0.03s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.194
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.664
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.006
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.209
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.207
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.208
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.396
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.440
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.475
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.433

Also, these words were printed out.

> INFO:tensorflow:Saving dict for global step 178: DetectionBoxes_Precision/mAP = 0.19424734, DetectionBoxes_Precision/mAP (large) = 0.20710744, DetectionBoxes_Precision/mAP (medium) = 0.2091659, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = 0.6642149, DetectionBoxes_Precision/mAP@.75IOU = 0.005652994, DetectionBoxes_Recall/AR@1 = 0.208, DetectionBoxes_Recall/AR@10 = 0.396, DetectionBoxes_Recall/AR@100 = 0.44, DetectionBoxes_Recall/AR@100 (large) = 0.43333334, DetectionBoxes_Recall/AR@100 (medium) = 0.475, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/classification_loss = 5.17783, Loss/localization_loss = 2.4297307, Loss/regularization_loss = 0.23962235, Loss/total_loss = 7.8471913, global_step = 178, learning_rate = 0.004, loss = 7.8471913
I0429 22:35:39.247992 140074595292928 estimator.py:2033] Saving dict for global step 178: DetectionBoxes_Precision/mAP = 0.19424734, DetectionBoxes_Precision/mAP (large) = 0.20710744, DetectionBoxes_Precision/mAP (medium) = 0.2091659, DetectionBoxes_Precision/mAP (small) = -1.0, DetectionBoxes_Precision/mAP@.50IOU = 0.6642149, DetectionBoxes_Precision/mAP@.75IOU = 0.005652994, DetectionBoxes_Recall/AR@1 = 0.208, DetectionBoxes_Recall/AR@10 = 0.396, DetectionBoxes_Recall/AR@100 = 0.44, DetectionBoxes_Recall/AR@100 (large) = 0.43333334, DetectionBoxes_Recall/AR@100 (medium) = 0.475, DetectionBoxes_Recall/AR@100 (small) = -1.0, Loss/classification_loss = 5.17783, Loss/localization_loss = 2.4297307, Loss/regularization_loss = 0.23962235, Loss/total_loss = 7.8471913, global_step = 178, learning_rate = 0.004, loss = 7.8471913

And it started training again.
> I0429 22:36:57.014948 140074595292928 basic_session_run_hooks.py:260] loss = 3.932395, step = 201 (362.309 sec)
INFO:tensorflow:global_step/sec: 0.297068
I0429 22:42:33.637230 140074595292928 basic_session_run_hooks.py:692] global_step/sec: 0.297068
INFO:tensorflow:loss = 3.5256753, step = 301 (336.623 sec)
I0429 22:42:33.637706 140074595292928 basic_session_run_hooks.py:260] loss = 3.5256753, step = 301 (336.623 sec)

After training 350 times, the above process was repeated the same way.

Is it right to be printed like this?
And I want to know the meaning of output.
